specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.2-preview.1/TextAnalytics.json

```diff
@@ -0,0 +1,3118 @@
+          "$ref": "#/definitions/TextAnalyticsError"
+        }
+      }
+    },
+    "ErrorResponse": {
+      "type": "object",
+      "required": [
+        "error"
+      ],
+      "properties": {
+        "error": {
+          "type": "object",
+          "description": "Document Error.",
+          "$ref": "#/definitions/TextAnalyticsError"
+        }
+      }
+    },
+    "TextAnalyticsError": {
+      "type": "object",
+      "required": [
+        "code",
+        "message"
+      ],
+      "properties": {
+        "code": {
+          "type": "string",
+          "enum": [
+            "InvalidRequest",
+            "InvalidArgument",
+            "InternalServerError",
+            "ServiceUnavailable",
+            "NotFound"
+          ],
+          "x-ms-enum": {
+            "name": "ErrorCodeValue",
+            "modelAsString": true
+          },
+          "description": "Error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "Error message."
+        },
+        "target": {
+          "type": "string",
+          "description": "Error target."
+        },
+        "innererror": {
+          "$ref": "#/definitions/InnerError",
+          "description": "Inner error contains more specific information."
+        },
+        "details": {
+          "type": "array",
+          "description": "Details about specific errors that led to this reported error.",
+          "items": {
+            "$ref": "#/definitions/TextAnalyticsError"
+          }
+        }
+      }
+    },
+    "TextAnalyticsWarning": {
+      "type": "object",
+      "required": [
+        "code",
+        "message"
+      ],
+      "properties": {
+        "code": {
+          "type": "string",
+          "enum": [
+            "LongWordsInDocument",
+            "DocumentTruncated"
+          ],
+          "x-ms-enum": {
+            "name": "WarningCodeValue",
+            "modelAsString": true
+          },
+          "description": "Error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "Warning message."
+        },
+        "targetRef": {
+          "type": "string",
+          "description": "A JSON pointer reference indicating the target object."
+        }
+      }
+    },
+    "ExtractiveSummarizationTask": {
+      "properties": {
+        "parameters": {
+          "properties": {
+            "model-version": {
+              "default": "latest",
+              "type": "string"
+            },
+            "loggingOptOut": {
+              "default": true,
+              "type": "boolean"
+            },
+            "stringIndexType": {
+              "$ref": "#/definitions/StringIndexType"
+            },
+            "sentenceCount": {
+              "default": 3,
+              "type": "integer"
+            },
+            "sortBy": {
+              "default": "Offset",
+              "enum": [
+                "Offset",
+                "Rank"
```

Comment:
are we sure capitalization is correct for these values? that burned me in testing so many times lol

Reply:
Since this is represented as an enum in C#, and the values are passed in the body rather than query params (as in our sync services), then I think these values should be capitalized.  I was running into issues without capitalization when testing with Analyze, so that's why I changed this.  Is this consistent with other parameter values for other Analyze tasks?

Reply:
Capitalization of first letter of enum values is correct.

Reply:
I confirm it works correctly in PPE, thanks!